mscore/mu3paletteadapter.cpp

```diff
@@ -133,6 +145,299 @@ void MU3PaletteAdapter::applyDrop(Ms::Score* score, Ms::Element* target, Ms::Ele
     }
 }
 
+//! NOTE Copied from Palette::applyPaletteElement
+bool MU3PaletteAdapter::applyPaletteElement(Ms::Element* element, Qt::KeyboardModifiers modifiers)
+{
+    Score* score = mscore->currentScore();
+    if (score == 0) {
```

Comment:
nullptr please

Reply:
missed note. do not pay attention to it.

Reply:
I transferred this code as is and it will be deleted soon, so it doesn't matter.
But it is important in those classes that are in the directories `framework` and `mu4` :)